tests/foreman/ui/test_host.py

```diff
@@ -956,6 +958,182 @@ def test_positive_search_by_org(self):
                 )
             )
 
+    @run_only_on('sat')
+    @tier2
+    def test_positive_sort_by_name(self):
+        """Create some Host entities and sort them by name ascendingly and then
+        descendingly
+
+        :id: 12f75ef9-23e6-48be-80ed-b354e8ac212b
+
+        :expectedresults: Host entities are sorted properly
+
+        :CaseImportance: High
+
+        :BZ: 1268085
+
+        :CaseLevel: Integration
+        """
+        org = entities.Organization().create()
+        alpha_name_list = list(string.ascii_lowercase[5::-1])
+        name_list = [
+            s + gen_string('alpha', 20).lower() for s in alpha_name_list]
+        for name in name_list:
+            entities.Host(name=name, organization=org).create()
+        with Session(self) as session:
+            set_context(session, org=org.name)
+            self.hosts.navigate_to_entity()
+            sorted_list_asc = self.hosts.sort_table_by_column('Name')
+            self.assertEqual(
+                [el.split('.', 1)[0] for el in sorted_list_asc],
+                name_list[::-1]
+            )
+            sorted_list_desc = self.hosts.sort_table_by_column('Name')
+            self.assertEqual(
+                [el.split('.', 1)[0] for el in sorted_list_desc],
+                name_list
+            )
```

Comment:
same as above

Reply:
that way we do not need alpha_name_list any more only name list
```python
name_list = [gen_string('alpha', 20) for _ in range(names_count)]

```

Reply:
Recap:
```python
names_count=5
name_list = [gen_string('alpha', 20) for _ in range(names_count)]
for name in name_list:
    entities.Host(name=name, organization=org).create()
with Session(self) as session:
   set_context(session, org=org.name)
   self.hosts.navigate_to_entity()
   sorted_list_asc = self.hosts.sort_table_by_column('Name')
   self.assertEqual(sorted_list_asc, sorted(sorted_list_asc))
   sorted_list_desc = self.hosts.sort_table_by_column('Name')
   self.assertEqual(sorted_list_desc, sorted(sorted_list_asc, reverse=True))
```

Reply:
what do you think?

Reply:
I have own reason and story why I implemented in current way, but I don't mind to change to your approach. Please, let me know if your review is more or less final and I will re-implement all methods and do full retest

Thanks for comments!

Reply:
@oshtaier I can understand your logic in the case where the names contains upper case/utf-8 chars and you do not want to deal with locales headache, but here you have lower case ASCII names.
Perhaps you should remove the lower() to have more general ASCII names?
In any case we can assert from these series of tests that the sorting is working only for the first char, and we cannot  assert that other chars in the strings are used for sorting.
Perhaps a more generic list should be used like:
```python
import six
name_list = sorted([gen_string('alpha', 20) for _ in range(names_count)], reverse=True, key=six.text_type.lower)
```
and your tests remain without modification except the name_list assignment, this should work with upper case and utf 8 too
what do you think? 
or I am missing some thing?